### PR TITLE
Updated testing documentation for running tests on payments

### PR DESCRIPTION
**Pull request by yachtcaptain23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,11 +44,18 @@ To bring up a dev environment, run `make docker-dev`.
 This brings up an additional vault service, used for integration testing of
 some auxiliary binaries.
 
-You can run the unit and integration tests via `make docker-test`
+### Testing
 
-Once you are in the Docker container you can run the web server via `go run bin/grant-server/main.go`.
+## Automated
+You can run the unit and integration tests via `make docker-test`. (Accurate?)
```
Comment (yachtcaptain23) [inline README.md:50]: Wasn't sure if `Automated` is the right terminology here. Maybe `Default Testing Behavior`?

File: README.md [hunk 2]
```diff
@@ -44,11 +44,18 @@ To bring up a dev environment, run `make docker-dev`.
 This brings up an additional vault service, used for integration testing of
 some auxiliary binaries.
 
-You can run the unit and integration tests via `make docker-test`
+### Testing
 
-Once you are in the Docker container you can run the web server via `go run bin/grant-server/main.go`.
```
Comment (yachtcaptain23) [inline README.md:49]: I don't think we actually need to run `go run /src/grant-server/main.go` like the documentation had expressed in the past. If so, I'll just remove 
```
`> go run /src/grant-server/main.go` # run this in the shell (Accurate? Do we need this?)
```

Comment (husobee) [reply README.md:49]: sounds good

File: README.md (hunk 1 above)
Comment (husobee) [reply README.md:50]: Default Testing Behavior sounds good.

**Review by husobee — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -44,11 +44,18 @@ To bring up a dev environment, run `make docker-dev`.
...
+## Automated
```
Comment (yachtcaptain23) [inline README.md:49]:
```suggestion
## Default Testing Behavior
```

File: README.md [hunk 4]
```diff
@@ -44,11 +44,18 @@ To bring up a dev environment, run `make docker-dev`.
...
-You can run the unit and integration tests via `make docker-test`
+### Testing
 
-Once you are in the Docker container you can run the web server via `go run bin/grant-server/main.go`.
+## Default Testing Behavior
+You can run the unit and integration tests via `make docker-test`. (Accurate?)
+
+## In Docker Container
+`make docker-dev` 
+`> go run /src/grant-server/main.go` # run this in the shell (Accurate? Do we need this?)
```
Comment (yachtcaptain23) [inline README.md:54]:
```suggestion
```

**Review by husobee — APPROVED** [review]

**Pull request merged**